test/extended/olm/olm.go

```diff
@@ -199,6 +200,16 @@ func archHasDefaultIndex(oc *exutil.CLI) bool {
 	return false
 }
 
+func marketplaceEnabled(oc *exutil.CLI) (bool, error) {
+	_, err := oc.AsAdmin().Run("get").Args("crds", "operatorhubs.config.openshift.io").Output()
```

Comment:
I think it would be to just check the clusterversion object and see if the "marketplace" capability is enabled in `status.capabilities`